src/vsr/replica.zig

```diff
@@ -1670,6 +1633,75 @@ pub fn Replica(
             return count;
         }
 
+        fn tally_message_and_receive_quorum_exactly_once(
+            self: *Self,
+            //prepare: *Prepare,
```

Comment:
What are your thoughts on taking the `prepare` here as an arg? Was it to double-check the `prepare_ok.context` against the `prepare.checksum`?

Reply:
Oops, that is old code. Yes, it originally was to double-check the prepare. But then I changed `start_view_change_from_other_replicas` and `nack_prepare_from_other_replicas` from `QuorumMessages` to `QuorumCounter`, so the function has to be more general.